remix-tests/tests/examples_4/SafeMath_test.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity ^0.4.24;
+import "./tests.sol";
```

Comment:
can this be changed to `import "./remix_tests.sol"` . This is how remix IDE inject the test framework.
@iurimatias would that be ok?